public/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+    "theme_color": "#f69435",
+    "background_color": "#f69435",
```

Comment:
change the theme color to our project theme color.

check our project theme color https://github.com/priyankarpal/ProjectsHut/blob/2e2bb6027a23b88564863a6ea0fc5fe40404a77c/tailwind.config.js#L23